File: per9/IO.cpp
#include <iostream>
using namespace std;

int main(){
    char nama_barang[50];
    int harga;
    int jumlah;

    cout << "Masukkan nama barang: ";
    cin >> nama_barang;
    cout << "Masukkan harga barang: ";
    cin >> harga;
    cout << "Masukkan jumlah barang: ";
    cin >> jumlah;
    int total = harga * jumlah;
    double diskon;

    if(total > 53100){
        diskon = total * 0.1;
    }else {
        diskon = 0;
    }
    int bayar = total - diskon;
    
    cout << "\n=====struk pembelian Jokomart=====" << endl;
    cout << "Nama Pembeli : " << nama_barang << endl;       
    cout << "Harga Barang : Rp. " << harga << endl;
    cout << "Jumlah Barang : " << jumlah << endl;  
    cout << "Total Harga : Rp. " << total << endl;
    cout << "Diskon : Rp. " << diskon << endl;  
    cout << "Total Bayar : Rp. " << bayar << endl;
    cout << "===============================" << endl;
    return 0;
}

File: per9/IO.py
nama=input("Nama Barang: ")
harga=float(input("Harga Barang : "))
jumlah=int(input("Jumlah Barang : "))


total=harga*jumlah  
if total > 53100:
    diskon = total * 0.01
else:
    diskon = 0

bayar = total - diskon

print("=====struk pembelian Jokomart=====")
print("Nama Pembeli :", nama)
print("Harga Barang : Rp.", harga)
print("Jumlah Barang :", jumlah)
print("Total Harga : Rp.", total)
print("Diskon : Rp.", diskon)
print("Total Bayar : Rp.", bayar)
print("=====================================")